pkg/operator/ceph/object/user/controller.go

```diff
@@ -334,6 +335,47 @@ func generateUserConfig(user *cephv1.CephObjectStoreUser) admin.User {
 		Keys:        make([]admin.UserKeySpec, 1),
 	}
 
+	if user.Spec.MaxBuckets != 0 {
+		userConfig.MaxBuckets = &user.Spec.MaxBuckets
+	}
+
+	if user.Spec.MaxObjects != 0 || user.Spec.MaxSize != "" {
+		quotaEnable := true
+		var maxSize int64 = -1
+		var maxObjects int64 = -1
+		if user.Spec.MaxObjects >= 0 {
+			maxObjects = int64(user.Spec.MaxObjects)
+		}
+		if user.Spec.MaxSize != "" {
+			maxSizeInt, _ := resource.ParseQuantity(user.Spec.MaxSize)
```

Comment:
We should check the err, right? Or else an invalid value will cause unexpected behavior
```suggestion
			maxSizeInt, err := resource.ParseQuantity(user.Spec.MaxSize)
```